app/Http/Controllers/Controller.php

```diff
@@ -40,6 +40,9 @@ public function home()
             'sans-serif' => 'Sans Serif',
             'serif' => 'Serif',
         ];
+
+        $font_sizes = array_combine(range(8, 12), array_map(fn($i) => "{$i} px", range(8, 12)));
```

Comment:
this syntax is new to me, very javascript-y

Reply:
I might need to check that against some older PHP versions now that you mention it.

Reply:
we can control the php version on this service so it should be fine if it's bleeding-edge

Reply:
Derp. any idea what version we are currently running?

Reply:
pdf is currently on `8.2` but it's easy to change if needed